style.css

```diff
@@ -1,224 +1,233 @@
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100&family=Poppins:wght@100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
 
-* {
-  box-sizing: border-box;
-}
-
-body {
+.body {
```

Comment:
- [ ] I believe you have a typo here. I think you wanted to select the HTML tag `body` instead of the class `.body` (notice the dot). your `body` still has a `margin: 8px` and this is because this style isn't get applied to the body. Please fix this typo so your design matches the Figma perfectly 👍.